packages/usa-alert/src/styles/_usa-alert.scss

```diff
@@ -2,15 +2,6 @@
 
 @use "uswds-core" as *;
 
-// Alert variables ---------- //
-$alert-icons: (
-  success: "check_circle",
-  warning: "warning",
-  error: "error",
-  info: "info",
-  emergency: "error",
-);
-
```

Comment:
Moved these icons to `alert-status-styles.scss` so that both alert and site alert have the same variants. This enables us to recommend users use the site alert markup structure for alerts that should share site margins.

Reply:
Resolving this comment because this change was reverted in [d3a2a08](https://github.com/uswds/uswds/pull/5636/commits/d3a2a083b4adb5545711acaccfcbaeeaaf278c82)